src/integrations/admin/activation-indexation-integration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Admin;
+
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast\WP\SEO\Conditionals\Traits\Admin_Conditional_Trait;
+use Yoast\WP\SEO\Integrations\Cleanup_Integration;
+
+/**
+ * This integration registers a run of the cleanup routine whenever the plugin is activated.
+ */
+class Activation_Indexation_Integration implements Integration_Interface {
```

Comment:
Let's have a more generic name here that doesn't include `Indexation` so that we can always use it for `upon activation` stuff

Reply:
I am conflicted about this one, since I don't want to introduce classes that do 20 completely different things, I would then rather move this integration to a activation folder that have 1 big uber integration for everything upon activation.

Reply:
Hm, that's fair point 🤔 I have no issue keeping it like this then, it does make sense to keep this class small :)

Reply:
Maybe `Indexation` is not reflective enough though? Maybe `Activation_Cleanup_Integration` could be better?